### ISSUE TITLE
Error when using route requirements

### ISSUE THREAD

**Issue by aszenz** [BolZer/symfony-typescript-routes#14]
A union type is generated for something like this in requirements

```php
#[Route(path: '/{id}', requirements: ['id' => '\d+|new'], options: ['expose' => true])]
```
```ts
{id: '\d+'|'new'}
```

This is of course not correct, it should be `int|'new'` instead, the real problem is that route requirements can be any regex, so I'm not sure if there's a way to correctly parse them for creating types.

Maybe it was better earlier with everything as string.

**Comment by BolZer** [comment]
@aszenz Will be fixed in #15 

The Union Parts (i. E. \d+ and 'new') will be individually "guessed". The previous implementation was a bit too naive.

**Comment by BolZer** [comment]
@aszenz V. 1.4.2 is released. This should fix your issue.
